triphut-specs/Spec/Vote/ContextValidator.hs

```diff
@@ -18,33 +30,119 @@ import Plutus.Model (
  )
 import Plutus.Model.V2 (
   DatumMode (InlineDatum),
-  logInfo,
   payToScript,
   refInputInline,
  )
 import Plutus.V1.Ledger.Interval (from)
-import PlutusTx.Prelude (($))
 import Spec.SpecUtils (amountOfAda)
 import Spec.Tally.Script (tallyNftTypedValidator)
-import Spec.Tally.Transactions (runInitTallyConfig, runInitTallyWithEndTimeInPast)
+import Spec.Tally.Transactions (
+  runInitTallyConfig,
+  runInitTallyWithEndTimeInFuture,
+  runInitTallyWithEndTimeInPast,
+ )
 import Spec.Tally.Utils (findTally, findTallyConfig)
 import Spec.Values (dummyTallyValue, dummyVoteValue)
 import Spec.Vote.Script (voteTypedValidator)
-import Spec.Vote.Transactions (runInitVote, runInitVoteConfig)
+import Spec.Vote.Transactions (runInitVote, runInitVoteConfig, runInitVoteMinterConfig, runInitVoteWithUser)
 import Spec.Vote.Utils (findVote, findVoteConfig)
-import Triphut.Vote (VoteActionRedeemer (Count))
-import Prelude (mconcat, pure, show, (*), (+), (<>))
+import Triphut.Vote (VoteActionRedeemer (Cancel, Count))
```

Comment:
This should be changes to `dao`

Reply:
They're changed to dao everywhere now @Randy1Burrell